lib/workers/evm_server.rb

```diff
@@ -31,4 +33,149 @@ def set_process_title
   def self.start(*args)
     new.start
   end
+
+  def start_server_environment(server)
```

Comment:
is it ok if `server` is nil here?  I can be the first time, before seed_primordial is run.